gormschema/gorm.go

```diff
@@ -187,3 +224,49 @@ func WithJoinTable(model any, field string, jointable any) Option {
 		})
 	}
 }
+
+func indirect(t reflect.Type) reflect.Type {
+	for t.Kind() == reflect.Ptr {
+		t = t.Elem()
+	}
+	return t
+}
+
+type (
+	// ViewOption configures a viewBuilder.
+	ViewOption func(*viewBuilder)
+	// ViewDefiner defines a view.
+	ViewDefiner interface {
+		ViewDef(dialect string) []ViewOption
+	}
+	viewBuilder struct {
+		db         *gorm.DB
+		createStmt string
+		// viewName is only used for the BuildStmt option.
+		// BuildStmt returns only a subquery; viewName helps to create a full CREATE VIEW statement.
+		viewName string
+	}
+)
+
+// CreateStmt accepts raw SQL with args to create a CREATE VIEW statement.
+func CreateStmt(sql string, args ...any) ViewOption {
+	return func(b *viewBuilder) {
+		b.createStmt = b.db.ToSQL(func(tx *gorm.DB) *gorm.DB {
+			return tx.Exec(sql, args...)
+		})
+	}
+}
```

Comment:
```suggestion
func CreateStmt(sql string, args ...any) ViewOption {
	return func(b *viewBuilder) {
		b.createStmt = b.db.ToSQL(func(tx *gorm.DB) *gorm.DB {
			return tx.Exec(sql, args...)
		})
	}
}
```

See my comment above. 
```suggestion
func CreateStmt(stmt string) ViewOption {
	return func(b *viewBuilder) {
		b.createStmt = b.db.ToSQL(func(tx *gorm.DB) *gorm.DB {
			return tx.Exec(stmt)
		})
	}
}
```